src/Storages/MergeTree/RPNBuilder.cpp

```diff
@@ -396,6 +396,12 @@ size_t RPNBuilderFunctionTreeNode::getArgumentsSize() const
 
 RPNBuilderTreeNode RPNBuilderFunctionTreeNode::getArgumentAt(size_t index) const
 {
+    size_t total_arguments = getArgumentsSize();
+    if (index >= total_arguments) /// Bug #52632
```

Comment:
```suggestion
    if (unlikely(index >= total_arguments)) /// Bug #52632
```

Reply:
Not an unreasonable request per se, but none of the code is performance-critical (e.g. in a hot loop). I'd prefer to not clutter the code and keep it as is.

Btw., the "native" alternative to likely/unlikely would be `[[likely]]` / `[[unlikely]]` ([C++20 docs](https://en.cppreference.com/w/cpp/language/attributes/likely)).